index.html

```diff
@@ -1090,36 +1087,74 @@ <h3>Supporting algorithms</h3>
     </p>
   </section>
   <section>
-    <h3>Data delivery</h3>
+    <h3>Data Collection and Delivery</h3>
     <p>
-      [=Data delivery=] from a [=platform collector=] can be activate and deactivated in an
+      [=Data collection=] from a [=platform collector=] can be activated and deactivated in an
       [=implementation-defined=] manner per [=source type=] and [=global object=].
     </p>
     <aside class="note">
-      It is recommended that the [=platform collector=] suspends low-level data polling
-      when there is no active [=data delivery=] to any {{PressureObserver}} [=relevant global object=].
+      It is recommended that the [=platform collector=] suspends low-level data collection
+      when there is no active [=data collection=] to any {{PressureObserver}}'s [=relevant global object=].
     </aside>
     <p>
-      The <dfn>data delivery</dfn> steps that are run when
-      an [=implementation-defined=] |data| sample of [=source type=] |source:PressureSource| is
-      obtained from [=global object=] |relevantGlobal|'s [=platform collector=],
-      are as follows:
+      The <dfn>data collection</dfn> steps given |relevantGlobal| as input, are run when
```

Comment:
One thing that's just occurred to me is that these steps make a virtual pressure source override a real source even a platform collector is currently fetching real platform data (i.e. if you call `PressureObserver.observe()` first, then call create+update virtual pressure source, the data will suddenly start coming from a different source).

This is different from what Generic Sensor does, but it's also valid (although the implementation would have to change a bit too). I'm just double checking if this has been done by design.

Reply:
I've attempted to do this in c01a9a913d9cc0d85b9043f5eefe5924b67fd40e